app/controllers/application_controller.rb

```diff
@@ -367,6 +367,7 @@ def report_data
     if options.nil? || options[:view].nil?
       model_view = process_params_model_view(params, options)
       @edit = session[:edit]
+      options_from_session(options)
```

Comment:
`get_view` is called twice because there are still some loose ends which call querying twice
1) when screen is rendered
2) when `report_data` is called

We should find out how to remove the first call, since it is abundant BUT, some screens stores some where clauses in `session[:options]`, these where clause are incorrect when `report_data` is called. You see some where clauses depends on selected tree node, some on model and some are stored in session based on some deep logic and I was unable to locate them, hence I did not removed `get_view` method.

To avoid using session stored where clause, I'd recommend finding how is the where clause which is used for this filtering generated and store it in options in https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller.rb#L286